drivers/dma/dma_stm32u5.c

```diff
@@ -405,6 +405,26 @@ static int dma_stm32_configure(const struct device *dev,
 		return -EINVAL;
 	}
 
+	if (config->source_burst_length > 64) {
```

Comment:
Define a constant for this:
```c
#define STM32U5_DMA_MAX_BURST_LEN 64 /* Max number of beats in a burst */
```
```suggestion
	if (config->source_burst_length > STM32U5_DMA_MAX_BURST_LEN) {
```